Swiftfin/Components/HTMLFormattedText.swift

```diff
@@ -0,0 +1,62 @@
+//
+// Swiftfin is subject to the terms of the Mozilla Public
+// License, v2.0. If a copy of the MPL was not distributed with this
+// file, you can obtain one at https://mozilla.org/MPL/2.0/.
+//
+// Copyright (c) 2024 Jellyfin & Jellyfin Contributors
+//
+
+import SwiftUI
+
+struct HTMLFormattedText: UIViewRepresentable {
+    let text: String
+    private let textView = UITextView()
+
+    init(_ content: String) {
+        self.text = content
+    }
+
+    func makeUIView(context: UIViewRepresentableContext<Self>) -> UITextView {
+        textView.widthAnchor.constraint(equalToConstant: UIScreen.main.bounds.width).isActive = true
```

Comment:
`UITextView`s in SwiftUI are very tricky to get right. I actually just helped someone make a `UITextView` wrapper so this came at the right time. For sizing we shouldn't set constraints and instead have to go to `setContentCompressionResistancePriority`.

Here is the code that I helped the other person with, so you'll have to decipher it a bit, but it is an example of how to wrap a `UITextView` in SwiftUI. I am unsure how it works with HTML or Markdown, so you will have to test that. This work may be bigger than you expected.

- https://pastebin.com/nbiTbwcN